hack/cluster-migration/main.go

```diff
@@ -82,7 +83,7 @@ func main() {
 	var config Config
 	flag.StringVar(&config.configPath, "config", "../../config/prow/config.yaml", "Path to prow config")
 	flag.StringVar(&config.jobConfigPath, "job-config", "../../config/jobs", "Path to prow job config")
-	// flag.StringVar(&config.jobConfigPath, "job-config", "../../config/jobs", "Path to prow job config")
+	flag.StringVar(&config.repo, "repo", "", "Find eligible jobs for a specific repo")
```

Comment:
if you start doing reporting and filtering you'll end with hundreds of flags , have you considered to take a different approach so this tool can be piped to another tools like `jq` per example, in the most unix style , so you generate the json output or csv and you postprocess the result

This will make this tool more sustainable

Reply:
Thank you for the feedback and I think adding a csv or json output would be a very good improvement. Can you help me understand what you're envisioning a bit more? Currently the tool has three primary workflows.
1. obtain general stats across all jobs. Things like total jobs, jobs per cluster, community vs non community jobs.
2. stats across all repositories. How many job each repo has, and how many are eligible to move to the community clusters.
3. (which this PR introduces) detailed repository view that shows the name of all jobs and whether the job is done, eligible to move to the community clusters, or not eligible to move. 

With the current structure, I think it would be quite easy to allow the user to specify the output format they're interested for any one of those workflows.

Does changing the output format of each of those workflows align with your vision. Or, are you thinking more generally that this tool would generate one significant output that's some form of combination of all three workflows? Or something else?

Reply:
@aojea , I rewrote a significant portion of the tool this weekend so all the data is normalized to a single hierarchical struct which can be dumped in its entirety using `--output json`. For now I'm continuing to keep the the default output but that may also evolve in time.